worker/src/RTC/RtpStreamSend.cpp

```diff
@@ -419,13 +446,13 @@ namespace RTC
 		auto seq          = packet->GetSequenceNumber();
 		auto* storageItem = this->storageItemBuffer.Get(seq);
 
+		this->ClearOldPackets(packet);
```

Comment:
Why moving out of `if` that is below? Cleanup is not needed unless that check and timestamps check below is `true`

Reply:
It is, it is: https://github.com/versatica/mediasoup/blob/v3/worker/src/RTC/SimulcastConsumer.cpp#L876

Reply:
I'm just being super cautious. `ClearOldPackets` will just return right away in that case, without much processing.

Reply:
Also, if not done here we would have to call it from two different places. IMHO it's cleaner having it here. We can move it if we find performance penalties.